client/app/telecom/telephony/alias/special/transfer/telecom-telephony-alias-special-transfer.html

```diff
@@ -0,0 +1,162 @@
+<section class="telephony-group-alias-special-transfer">
+
+    <header>
+        <section-back-link section-back-link-to-state="telecom.telephony.alias.special"
+                           section-back-link-title="{{ 'telephony_alias_special_transfer_title_back_link' | translate }}">
+        </section-back-link>
+        <h1 data-translate="telephony_alias_special_transfer_title"></h1>
+    </header>
+
+
+    <div class="telecom-section-content">
+
+        <toast-message></toast-message>
+
+        <div class="row">
+            <div class="col-md-6 widget-presentation"
+                 data-ng-if="!AliasSpecialTransferCtrl.isLoading">
+                <div class="alert alert-info"
+                     role="alert">
+                    <span data-translate="telephony_alias_special_transfer_description"></span>
+                    <a class="btn btn-link p-0"
+                       ui-sref="telecom.telephony.alias.special.fees"
+                       data-translate="telephony_alias_special_transfer_description_link"></a>
+                </div>
+                <table class="table">
```

Comment:
+`table-hover` (any reason to not wrap your actual `<table>` with `.table-responsive` class?)